Dockerfile

```diff
@@ -43,12 +43,12 @@ RUN make package build compress
 
 ## production ##################################################################
 
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
[Skopeo](https://github.com/containers/skopeo/blob/main/install.md) doesn't provide a direct binary to download. Its standard distribution method is via package managers and is only available in ubuntu 20.10+. I bumped to 22 since it's the newest version, and based on our tests, no bugs were introduced. Better to keep stuff updated